### PR TITLE
Change CircleCI environment to Kubernetes 1.21.3

### PR DESCRIPTION
**Pull request by mkysel**
- change the default k8s version for all tests
- add some useful debugging to failed admin pod startup in test
- bump nuodb admin pod readiness timeout to 5 sec
- added `-c <container>` to all invocations in order to avoid kubectl warnings
- change kubeconfig chmod to avoid warnings

### CODE REVIEW

File: scripts/ci/install_deps.sh [hunk 1]
```diff
@@ -15,7 +15,10 @@ wget https://get.helm.sh/helm-"${HELM_VERSION}"-linux-amd64.tar.gz -O /tmp/helm.
 tar xzf /tmp/helm.tar.gz -C /tmp --strip-components=1 && chmod +x /tmp/helm && sudo mv /tmp/helm /usr/local/bin
 
 if [[ "$REQUIRES_MINIKUBE" == "true" ]]; then
-    # Download minikube.
+  sudo apt-get update
+  sudo apt-get install -y conntrack
```
Comment (adriansuarez) [inline scripts/ci/install_deps.sh:19]: Is this now needed by Minikube, or does it enable some diagnostics?

**Review by adriansuarez — APPROVED** [review]
Looks good.

File: scripts/ci/install_deps.sh (hunk 1 above)
Comment (mkysel) [reply scripts/ci/install_deps.sh:19]: by k8s

**Review by bkelly-ndb — APPROVED** [review]

**Review by sivanov-nuodb — APPROVED** [review]
Thanks!

**Pull request merged**